src/Microsoft.Dnx.Runtime/ApplicationHostContext.cs

```diff
@@ -76,6 +74,9 @@ public class ApplicationHostContext
                 {
                     NuGetDependencyProvider.ApplyLockFile(_lockFile);
 
+                    ProjectResolver = new LockFileBasedProjectResolver(_lockFile, ProjectDirectory, Project);
```

Comment:
You don't need to pass `ProjectDirectory` in because you have `Project.ProjectDirectory`